### PR TITLE
Updated sensor documentation to be more approachable to new users.

### PR DESCRIPTION
**Pull request by mwunderling**
Existing page doesn't provide a conceptual explanation and requires too much interpretation by the user.  Updated for readability, context, and written for a "new user" perspective.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @mwunderling, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17041).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/sensor.markdown [hunk 1]
```diff
@@ -9,13 +9,13 @@ ha_domain: sensor
...
+Sensors are a basic platform component in Home Assistant. They monitor the states and conditions of a variety of entities. An entity can be many things. This can include a physical device like a motion sensor that reports the battery level, a web service that retrieves the weather temperature, a built-in function that calculates the sun's elevation relative to your GPS position, or even a custom sensor you may have created to report the free space on your laptop. These are all *things* reporting different types of information.
 
-Home Assistant currently supports a wide range of sensors. They are able to display information which are provides by Home Assistant directly, are gathered from web services, and, of course, physical devices.
+Some of these sensors are built-in to Home Assistant, some are created automatically when you add an integration (see this [list](https://www.home-assistant.io/integrations/#sensor)), and some can be created manually. [Statistics](https://www.home-assistant.io/integrations/statistics) and [Template](https://www.home-assistant.io/integrations/template) sensors are two examples of the last case.
```
Comment (frenck) [inline source/_integrations/sensor.markdown]: Don't use full URLs within the same site.

```suggestion
Some of these sensors are built-in to Home Assistant, some are created automatically when you add an integration (see this [list](/integrations/#sensor)), and some can be created manually. The [Statistics](/integrations/statistics) and [Template](/integrations/template) sensors are two examples of the last case.
```

File: source/_integrations/sensor.markdown [hunk 2]
```diff
@@ -9,13 +9,13 @@ ha_domain: sensor
...
 
 ## Device Class
 
-The way these sensors are displayed in the frontend can be modified in the [customize section](/docs/configuration/customizing-devices/). The following device classes are supported for sensors:
+The type of data a sensor returns impacts how it is displayed in the frontend. This is controlled by the sensor's device class designation. Built-in sensors and many created from an integration will have this designation predefined. Those can be modified in the [customize section](https://www.home-assistant.io/docs/configuration/customizing-devices/). When manually creating a new sensor the device class may be optionally assigned. A full list of available sensor device classes is below:
```
Comment (frenck) [inline source/_integrations/sensor.markdown]:
```suggestion
The type of data a sensor returns impacts how it is displayed in the frontend. This is controlled by the sensor's device class designation. Built-in sensors and many created from an integration will have this designation predefined. Those can be modified in the [customize section](/docs/configuration/customizing-devices/). When manually creating a new sensor the device class may be optionally assigned. A full list of available sensor device classes is below:
```

**Review by frenck — APPROVED** [review]
Thanks, @mwunderling 👍 

Great first contribution to our documentation ❤️

**Pull request merged**